interface-definitions/system-login.xml.in

```diff
@@ -8,6 +8,19 @@
           <priority>400</priority>
         </properties>
         <children>
+          <leafNode name="autologout">
```

Comment:
I wonder if we should rename this to timeout?

Reply:
Done!